### DISCUSSION TITLE
Using sentry for error monitoring

### DISCUSSION THREAD

**Discussion by adieyal** [adieyal/dynamicprompts#33] [Ideas]
@akx what are your thoughts on using Sentry or some other error monitoring service? Opt-in by default

**Comment by akx** [comment]
I don't think that's feasible for a library, tbh. There's a whole bunch of privacy concerns - for prior art, you can look at all the backlash there's been for just basic usage telemetry for a bunch of libraries...

**Reply by adieyal** [reply]
I can understand hidden telemetry or opt-out but surely opt-in, in an open source project is a different story? Also, I should probably have posted this on the extension project - that's where the hard to diagnose issues usually pop up.

**Reply by akx** [reply]
Sorry, I missed "opt-in by default". Still... I'd rather go with a nice issue template for bug reporting on sd-dynamic-prompts -- or, I suppose, just not responding to issues that can't be reproduced.
